furusiya/model/weapons/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .bow import Bow
```

Comment:
Relative imports are generally frowned upon. Please specify the full import name (sorry)